src/test_utils.rs

```diff
@@ -783,17 +808,35 @@ impl TestContext {
         Chat::load_from_db(&self.ctx, chat_id).await.unwrap()
     }
 
+    /// Returns 1:1 [`Chat`] with another account PGP-contact.
+    /// Panics if the chat does not exist.
+    ///
+    /// This first creates a contact, but does not import the key,
+    /// so may create a PGP-contact with a fingerprint
+    /// but without the key.
+    pub async fn get_pgp_chat(&self, other: &TestContext) -> Chat {
```

Comment:
Since PGP chats are the more common type of chat now, I think it would make sense to call this function `get_chat()` and the other one `get_email_chat()` or `get_unencrypted_chat()` or similar. But it's not that important, e.g. if this would mean that a lot of test have to be changed.

Reply:
create_chat() to create_pgp_chat()